ember-cli-build.js

```diff
@@ -1,9 +1,13 @@
 'use strict';
 
+const nodeSASS = require('node-sass');
```

Comment:
Is there a particular reason we prefer node-sass over the default ember-cli-sass configuration?

Reply:
(The default ember-cli-sass blueprint now adds a `sass` dependency and looks for that. The config added here is only necessary if we want to use a different implementation than that.)